.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
   - docker-compose -v
   - docker -v
   # symfony application folder
-  - mkdir symfony
+  - mkdir -p .data/src
```

Comment:
not a big fan of installing symfony alongside database information. Please, remove this line.